src/auth/screens/events.screen.js

```diff
@@ -380,7 +395,7 @@ class Events extends Component {
             style={styles.linkDescription}
             onPress={() => this.navigateToRepository(userEvent, true)}
           >
-            {userEvent.payload.forkee.full_name}
+            {userEvent.payload.forkee}
```

Comment:
After normalization, userEvent.payload.forkee is simply the repo name (which can be later found in entities.repo)